Update Giphy client; Fixes random button;

rafavalerio opened the pull request:
# To Do
- [x] Update the Giphy client used (`giphy-js-sdk-core` => `@giphy/js-fetch-api`)
- [x] Add a new `I'm Feeling EXTREMELY Lucky` button

# GIF
_This is using the new `EXTREMELY Lucky` button!_ 🤞 

<img src="https://media4.giphy.com/media/xT1Ra2pTK8zCsvf97O/giphy.gif" alt="fix" width="100%" />

nickspiel commented on src/components/QuickSearch.js:
```diff
@@ -1,10 +1,10 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled from 'styled-components'
+import React from "react"
```
Can you update your linter to keep the current code style?

nickspiel commented on src/components/QuickSearch.js:
```diff
@@ -14,9 +14,9 @@ const Button = styled.button`
-  border: solid 1px #74FAA2;
-  box-shadow: 2px 2px 0 0 #74FAA2;
-  color: #74FAA2;
+  border: solid 1px #74faa2;
```
Here too.

nickspiel reviewed (approved):
This is awesome! Thanks for the PR. All looks great, would you be able to roll back the linter changes to keep things consistent?

rafavalerio commented:
So it seems `.prettierrc` was set to double quotes, but the project was mostly using single quotes. I've updated to reflect that and updated everything.

As for the color's case, see this: https://github.com/prettier/prettier/issues/5158
Seems lowercase is the default these days, So I did a full update is that's ok :)

nickspiel commented:
Absolute legend! Really appreciate the update, the prettier configuration clearly needs a little more work to enforce the style.
